sci_calc_code/src/UIElements/InputBox.cpp

```diff
@@ -144,7 +144,10 @@ void InputBox::draw() {
     this -> cursor.setY(this -> y);
 }
 
-
+//For repeat deleting
```

Comment:
Does this part need to be global? How about moving this into the .h file as a variable of the class? This also makes the code nicer to read I think.

Reply:
Thank you for your suggestion. I have moved the variables `delPressedTime`, `lastDeleteTime`, and `repeatDelete` from being global to private member variables within the class as you recommended. This change is reflected in the [latest commit](https://github.com/shaoxiongduan/sci-calc/pull/12/commits/fb05ccbd515a692c205b74aa17e2e77dec1c13ca). I agree that this improves the readability and maintainability of the code by adhering to better encapsulation practices. 

Please let me know if there are any further improvements or adjustments needed.